src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <>
+    <h1> Under Construction</h1>
+  </>
+);
+
+export default Categories;
```

Comment:
- [ ] According to the project requirement, you should display a button with the text "Check status". Not the heading. Please implement it

Reply:
Hi @roshan-bajgain. Thanks for your quite clear feedback. Now the problem seem to be fixed.